images/ubuntu/19.04/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM docker.io/library/ubuntu:19.04
+
+ENV NAME=ubuntu-toolbox VERSION=19.04
+LABEL com.github.debarshiray.toolbox="true" \
+      com.redhat.component="$NAME" \
+      name="$FGC/$NAME" \
+      version="$VERSION" \
+      usage="This image is meant to be used with the toolbox command" \
+      summary="Base image for creating Ubuntu 19.04 toolbox containers" \
+      maintainer="Debarshi Ray <rishi@fedoraproject.org>"
```

Comment:
May I convince you to be the maintainer of these Ubuntu images? :)

Reply:
Probably not, I only use Centos and Fedora. Debian and its derivatives aren't anything I work with normally. I really just did this one as a demonstration.

Reply:
Ok.

Before we announce support for Ubuntu, we would need somebody who cares enough about Ubuntu to ensure that Toolbox keeps working on a day to day basis with Ubuntu hosts and images. I am a little apprehensive of taking up that responsibility myself. I could, but it seems wise to spread the load around a bit.

Reply:
Anyway, in the meantime, we can work towards removing the rough edges before thinking of making any announcements. :)

Reply:
Sounds good to me.